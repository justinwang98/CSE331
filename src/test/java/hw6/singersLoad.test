# Test the parser on a graph that contains several nodes

LoadGraph sing songs.tsv
ListChildren sing JayChou