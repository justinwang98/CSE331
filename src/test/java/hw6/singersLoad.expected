# Test the parser on a graph that contains several nodes

loaded graph sing
the children of JayChou in sing are: BTS(Chinese) JJLin(Chinese) JJLin(Taiwanese) JokerXue(Chinese) Lisa(Chinese) Twice(Taiwanese) WangLeeHom(Chinese) WangLeeHom(Taiwanese)